app/sdil/models/RegistrationFormData.scala

```diff
@@ -39,11 +39,33 @@ case class RegistrationFormData(rosmData: RosmRegistration,
                                 contactDetails: Option[ContactDetails] = None,
                                 smallProducerConfirmFlag: Option[Boolean] = None) {
 
+  lazy val isSmall: Boolean = {
+    total(packageOwn, copackedVolume) < 1000000
+  }
+
+  lazy val isCopacked: Boolean = {
+    copackedVolume.exists(_.total != 0)
+  }
+
+  lazy val isImporter: Boolean = {
+    imports.contains(true)
+  }
+
+  lazy val isCopacker: Boolean = {
+    packageCopack.nonEmpty
+  }
+
   lazy val isVoluntary: Boolean = {
-    total(packageOwn, copackedVolume) < 1000000 && copackedVolume.exists(_.total != 0) && imports.contains(false) && packageCopack.isEmpty
+     isSmall && isCopacked && !isImporter && !isCopacker
   }
 
-  private def total(pack: Option[Litreage], copack: Option[Litreage]) = pack.fold[BigDecimal](0)(_.total) + copack.fold[BigDecimal](0)(_.total)
+  lazy val isEntitledToSmallProducerExemption: Boolean = {
```

Comment:
Only used once so doesn't necessarily need to be a val although might be clearer since it is used somewhere completely different so I can let this one go I think